app/src/main/java/com/xinto/opencord/ui/screen/Chat.kt

```diff
@@ -176,8 +175,31 @@ private fun ChatScreenLoaded(
                                     onClick = {},
                                     onLongClick = { showBottomDialog = true }
                                 ),
+                            reply = message.isReply.let { if (it) it else null }?.letComposable { _ ->
```

Comment:
I've noticed this let chain in a few places and it's pretty ugly, what about an extension function that's combined like `ifComposable`

Reply:
Makes sense